scripts/beegfspkgs.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 #
+BEEGFS_VER=713
```

Comment:
You may change this to ${1-713} so that you can pass it as a parameter with a default value to 713